### ISSUE TITLE
Proxy image on comment ?

### ISSUE THREAD

**Issue by magsout** [webcompat/webcompat.com#201]
@miketaylr What do you think of a method for proxy images to reduce the size ?

Example : http://webcompat.com/issues/163

![capture decran 2014-07-19 a 20 32 41](https://cloud.githubusercontent.com/assets/1997108/3635459/1e3f15ca-0f73-11e4-83f5-ce06e2417294.png)

**Comment by miketaylr** [comment]
New plan, we save a small version and display that in our UI via Pillow (depends on #710), and link to the original.

**Comment by miketaylr** [comment]
Given that, let's close this.
